tpm_test.go

```diff
@@ -58,12 +59,25 @@ var (
 	testEncodedEkCertChain []byte
 )
 
+func decodeHexString(s string) []byte {
+	b, _ := hex.DecodeString(s)
```

Comment:
Hm, no error handling here, is that safe? If this is purely used in tests, should there be an "if err != nil { panic(err) }" here instead?

Reply:
I could probably do that, or pass in the *testing.T instance.

Reply:
I've updated this so that it calls testing.T.Fatal() if there's an error now.